dime-research-standards/pillar-4-data-security/data-security-resources/dime-data-nda.md

```diff
@@ -26,12 +26,10 @@ that may be revealed while performing the research tasks;
 (e.g., drives, tapes, transcripts),
 unless specifically requested to do so by the Principal Investigator;
 
-4. Keep all _personal data_ stored in any form or format (e.g., files, drives, tapes, transcripts) secure
-both when transferred to me, and while it is in my possession.
-Examples of personal data includes both straightforwardly identifying data,
-such as name, address, and personal identification number,
-as well as, less straightforwardly identifying data,
-such as location data, online identification data such as an internet protocol (IP) address and cookie identifiers, and physical characteristics.
+4. Keep all _personal data_ (as defined by the [World Bank Personal Data Directive](https://ppfonline.worldbank.org/search/9228ae8f-f14a-40a0-a8af-ec7c7d1d7885?ver=current))
```

Comment:
I think still useful to have examples, but pull those from the examples the privacy policy uses. The definition from the privacy policy FAQs is "“any information relating to an identified or identifiable individual. An identifiable individual is one who can be identified by reasonable means, directly or indirectly, by reference to an attribute or combination of attributes within the data, or combination of the data with other available information. Attributes that can be used to identify an identifiable individual include, but are not limited to, name, identification number, location data, online identifier, metadata and factors specific to the physical, physiological, genetic, mental, economic, cultural or social identity of an individual.” https://worldbankgroup.sharepoint.com/:w:/r/sites/wbsites/DPO/_layouts/15/Doc.aspx?sourcedoc=%7BDB268C90-E2FD-4540-A46B-CB333D46900C%7D&file=DP%20-%20FAQ.docx&action=default&mobileredirect=true&cid=05fcff1f-6c5e-46a2-a9f9-3ab5b5900c2e

Reply:
done!